apps/judicial-system/web/src/components/IndictmentCaseFilesList/IndictmentCaseFilesList.tsx

```diff
@@ -255,6 +256,28 @@ const IndictmentCaseFilesList: FC<Props> = ({
           <CaseFileTable caseFiles={uploadedCaseFiles} onOpenFile={onOpen} />
         </Box>
       )}
+      {showSubpoenaPdf &&
+        workingCase.defendants &&
+        workingCase.defendants.length > 0 && (
+          <Box marginBottom={5}>
+            <Text variant="h4" as="h4" marginBottom={1}>
+              {formatMessage(strings.subpoenaTitle)}
+            </Text>
+            {workingCase.defendants.map((defendant) => (
+              <Box marginBottom={2} key={`subpoena-${workingCase.id}`}>
```

Comment:
This will result in duplicate keys if there are multiple defendants. Did you mean `key={`subpoena-${defendant.id}`}`

Reply:
Good catch, will change.

Reply:
Your suggestion will work for now. Later on, we will have multiple subpoenas per defendant so we need to make further changes.